docs/Resources/resources.md

```diff
@@ -84,7 +88,3 @@ The NeuroML conversion of the [Virtual Worm Blender files](http://caltech.wormba
 ![CElegansnC](https://github.com/openworm/CElegansNeuroML/raw/master/CElegans/images/CElegans_nC.png)
 
 More information about working with the data within it and other data entities can be found on the data representation project page \<data-rep\>.
-
-### OpenWorm Spreadsheet data
-
-We keep a [publicly accessible archive of data sets](https://drive.google.com/#folders/0B_t3mQaA-HaMejlrMmpnR2VjN0U) that we have come across and adapted on Google Drive. We are currently in the process of consolidating these data into the OpenWorm database. More information about working with the data within it and other data entities can be found on [the data representation project page](/Projects/datarep/).
```

Comment:
I presume this should not be removed?

Reply:
It's fine -- this was merged into the owmeta section.